packages/http/src/forwarder/HttpForwarder.ts

```diff
@@ -39,6 +39,7 @@ export class HttpForwarder implements IForwarder<IHttpOperation, IHttpRequest, I
       statusCode: response.status,
       headers: response.headers,
       body: response.data,
+      responseType: (response.request && response.request.responseType) || '',
```

Comment:
Can really the request be undefined? This is really weird. I'd say it's a problem in the types.

Reply:
It's `any`, so well, might be a null.